src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -105,43 +105,42 @@ public DriveSubsystem() {
    *                    joystick, from -1 to 1.
    */
   public void drive(final double moveRequest, final double turnRequest) {
-    /**
-     * Since moveRequest is from -1 to 1 and we need a value in meters per second to
-     * feed to ChassisSpeeds, we scale the moveRequest to the speed limit as
-     * converted to ft/s.
-     */
-    final double moveReqScaled = org.usfirst.frc.team4999.utils.Utils.map(moveRequest, -1, 1,
-        -Units.feetToMeters(SPEED_LIMIT_FEET_PER_S), Units.feetToMeters(SPEED_LIMIT_FEET_PER_S));
-    /**
-     * Since turnRequest is from -1 to 1 and we need a value in radians per second
-     * to feed to ChassisSpeeds, we scale the turnRequest to the angular velocity
-     * limit in radians/second.
-     */
-    final double turnReqScaled = org.usfirst.frc.team4999.utils.Utils.map(turnRequest, -1, 1, -TURN_LIMIT_RAD_PER_S,
-        TURN_LIMIT_RAD_PER_S);
-    /**
-     * The object that handles the calculations for how fast each side of the robot
-     * should drive to accomplish the scaled move and turn requests.
-     */
-    final var chassisSpeeds = new ChassisSpeeds(moveReqScaled, 0, turnReqScaled);
-    /**
-     * The object that converts the overall chassis speed into individual wheel
-     * speeds.
-     */
-    final DifferentialDriveWheelSpeeds wheelSpeeds = kinematics.toWheelSpeeds(chassisSpeeds);
-    /**
-     * The variables that store the outputs of the wheel speeds, in meters per
-     * second.
-     */
-    final double leftMPerS = wheelSpeeds.leftMetersPerSecond;
-    final double rightMPerS = wheelSpeeds.rightMetersPerSecond;
-    /**
-     * The variables that store the wheel speeds in TalonFX encoder ticks per
-     * second, as converted from m/s to ft/s.
-     */
-    final double leftETPerS = feetToEncTicks(Units.metersToFeet(leftMPerS));
-    final double rightETPerS = feetToEncTicks(Units.metersToFeet(rightMPerS));
     if (pidEnabled) {
+      /**
+       * Since moveRequest is from -1 to 1 and we need a value in meters per second to
+       * feed to ChassisSpeeds, we scale the moveRequest to the speed limit as
+       * converted to ft/s.
+       */
+      final double speedLimitMs = Units.feetToMeters(SPEED_LIMIT_FEET_PER_S);
```

Comment:
If we're choosing SI units, be consistent and thorough about it.  Calling a method to convert a constant on every loop is wasteful.